package.json

```diff
@@ -29,14 +29,16 @@
     "test": "grunt test",
     "lint": "eslint Gruntfile.js tasks",
     "snyk-protect": "snyk protect",
-    "prepublish": "npm run snyk-protect"
+    "prepublish": "npm run snyk-protect",
+    "prepare": "npm run snyk-protect"
   },
   "dependencies": {
     "@mapbox/togeojson": "~0.16.0",
     "geobuf": "~3.0.1",
     "jsdom": "~15.0.0",
     "pbf": "~3.2.0",
-    "topojson-server": "~3.0.0"
+    "topojson-server": "~3.0.0",
+    "snyk": "^1.192.4"
```

Comment:
Hey @snyk-bot , why did you move your tool from `devDependencies` to `dependencies`?